### DISCUSSION TITLE
Welcome to cargo-crev Discussions!

### DISCUSSION THREAD

**Discussion by dpc** [crev-dev/cargo-crev#438] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KaiserKarel** [comment]
We're starting an effort to get every single dependency reviewed by our team/people we trust. Looking forward to using crev!

**Reply by dpc** [reply]
Please share how it went!
